index.ts

```diff
@@ -238,25 +267,25 @@ function ensureTypeScriptInstance(loaderOptions: LoaderOptions, loader: any): {
         compilerOptions.module = 0 /* None */;
         libFileName = 'lib.es6.d.ts';
     }
-    
+
     if (loaderOptions.transpileOnly) {
         // quick return for transpiling
         // we do need to check for any issues with TS options though
         var program = compiler.createProgram([], compilerOptions),
-            diagnostics = program.getOptionsDiagnostics 
-                ? program.getOptionsDiagnostics() 
-                : program.getCompilerOptionsDiagnostics();
+            diagnostics = program.getOptionsDiagnostics
+                ? program.getOptionsDiagnostics()
+                : languageService.getCompilerOptionsDiagnostics();
```

Comment:
```getCompilerOptionsDiagnostics``` doesn't exist on ```program```. Only on the language service object. It won't even compile like that. See ```typescript.d.ts```. Am I missing something here?